src/accidental.ts

```diff
@@ -17,23 +17,61 @@ import { Modifier } from './modifier';
 import { Glyph } from './glyph';
 import { GraceNoteGroup } from './gracenotegroup';
 import { GraceNote } from './gracenote';
+import { ModifierContextState } from './modifiercontext';
+import { Voice } from './voice';
+import { Note } from './note';
+import { StaveNote } from './stavenote';
+
+type Line = {
+  column: number;
+  line: number;
+  flatLine: boolean;
+  dblSharpLine: boolean;
+  numAcc: number;
+  width: number;
+};
+
+type AccidentalListItem = {
```

Comment:
Can you move these types into `format` where they are used? (type aliases are allowed inside function bodies)

Reply:
Line must stay out as it is used on other functions.